tests/test_plans/test_plan_rel_0.7-prod.md

```diff
@@ -0,0 +1,154 @@
+
+----
+## RELEASE TESTING
+
+As part of the release testing, following tests will be executed:
+- [Kruize Remote monitoring Functional tests](/tests/scripts/remote_monitoring_tests/Remote_monitoring_tests.md)
+- [Fault tolerant test](/tests/scripts/remote_monitoring_tests/fault_tolerant_tests.md)
+- [Stress test](/tests/scripts/remote_monitoring_tests/README.md)
+- [DB Migration test](/tests/scripts/remote_monitoring_tests/db_migration_test.md)
+- [Recommendation and box plot values validation test](https://github.com/kruize/kruize-demos/blob/main/monitoring/remote_monitoring_demo/recommendations_infra_demo/README.md)
+- [Scalability test (On openshift)](/tests/scripts/remote_monitoring_tests/scalability_test.md) - scalability test with 5000 exps / 15 days usage data
+- [Kruize remote monitoring demo (On minikube)](https://github.com/kruize/kruize-demos/blob/main/monitoring/remote_monitoring_demo/README.md)
+- [Kruize local monitoring demo (On openshift)](https://github.com/kruize/kruize-demos/blob/main/monitoring/local_monitoring_demo)
+- [Kruize local monitoring Functional tests](/tests/scripts/local_monitoring_tests/Local_monitoring_tests.md)
+
+
+| #  | TEST SUITE                                     | EXPECTED RESULTS                                       | ACTUAL RESULTS                                         | COMMENTS                                                                                                                                                                                                                                                                                                                                                                                                          |
+|----|------------------------------------------------|--------------------------------------------------------|--------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| 
+| 1  | Kruize Remote monitoring Functional testsuite  | TOTAL - 527, PASSED - 485 / FAILED - 42                | TOTAL - 527, PASSED - 485 / FAILED - 42                | Existing issues - [559](https://github.com/kruize/autotune/issues/559), [610](https://github.com/kruize/autotune/issues/610)                                                                                                                                                                                                                                                                                      |
```

Comment:
@shreyabiradar07 Please update the test results for PR 1627 as well. I had submitted a jenkins runs, pick the numbers from there.